monai/losses/aucm_loss.py

```
@@ -0,0 +1,142 @@
+        Args:
+            input: the shape should be B1HW[D], where the channel dimension is 1 for binary classification.
+            target: the shape should be B1HW[D], with values 0 or 1.
+
+        Returns:
+            torch.Tensor: scalar AUCM loss.
+
+        Raises:
+            ValueError: When input or target have incorrect shapes.
+        """
+        if input.shape[1] != 1:
+            raise ValueError(f"Input should have 1 channel for binary classification, got {input.shape[1]}")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)